course-mw/enrolment-actor/src/main/scala/org/sunbird/enrolments/CourseEnrolmentActor.scala

```diff
@@ -44,6 +44,8 @@ class CourseEnrolmentActor @Inject()(@Named("course-batch-notification-actor") c
     var userCoursesDao: UserCoursesDao = new UserCoursesDaoImpl()
     var batchUserDao  : BatchUserDao   = new BatchUserDaoImpl()
     var groupDao: GroupDaoImpl = new GroupDaoImpl()
+    val isAllEnrolemts = if (StringUtils.isNotBlank(ProjectUtil.getConfigValue("is_all_enrolments")))
```

Comment:
change the variable names as follows:
isAllEnrolments -> isRetiredCoursesIncludedInEnrolList
is_all_enrolments -> enrolment.list.include.retired.courses

Also, add the below line in externalresources.properties file,
enrolment.list.include.retired.courses=false